### PR TITLE
Fix service worker errors

### PR DESCRIPTION
**Pull request by tonylepmets**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ yarn release
 
 ## Changelog
 
+### 6.88.2
```
Comment (dijs) [inline README.md]: Bit of a version bump, don't ya think?

File: packages/react-storefront/src/router/serviceWorker.js [hunk 2]
```diff
@@ -120,19 +129,23 @@ export async function clearCache() {
  * @private
  */
 export async function waitForServiceWorkerController() {
-  if (!navigator.serviceWorker || !navigator.serviceWorker.ready) {
```
Comment (dijs) [inline packages/react-storefront/src/router/serviceWorker.js]: Why was this removed?

File: packages/react-storefront/src/router/serviceWorker.js [hunk 3]
```diff
@@ -120,19 +129,23 @@ export async function clearCache() {
...
-    navigator.serviceWorker.ready.then(() => {
-      if (navigator.serviceWorker.controller) {
-        return resolve(true)
-      }
-      navigator.serviceWorker.addEventListener('controllerchange', () => {
-        return resolve(true)
+    navigator.serviceWorker.ready
+      .then(() => {
+        if (navigator.serviceWorker.controller) {
+          resolve(true)
```
Comment (dijs) [inline packages/react-storefront/src/router/serviceWorker.js]: This needs to be returned

File: packages/react-storefront/src/router/serviceWorker.js (hunk 2 above)
Comment (tonylepmets) [reply packages/react-storefront/src/router/serviceWorker.js]: `navigator.serviceWorker.ready` immediately rejects in the FF setting case.

**Comment by tonylepmets** [comment]
Updated @dijs

File: README.md [hunk 4]
```diff
@@ -74,6 +74,25 @@ yarn release
 
 ## Changelog
 
+### 6.71.1
+
+- Fixed unhandled exceptions relating to service workers in Firefox and Safari.
+
+### 6.71.0
```
Comment (dijs) [inline README.md]: I had to revert these changes from master, could you please remove the stale-while-revalidate change log entry

Comment (markbrocato) [reply README.md]: done
